### PR TITLE
Add processor architecture to distribution request

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

Add processor architecture to distribution request

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Distribution_Request.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ assignees: ''
 
 - Name of the Distribution: 
 - Version of the Distribution:
+- Processor Architecture (One per request):
```
Comment (iSazonov) [inline .github/ISSUE_TEMPLATE/Distribution_Request.md]: Maybe add link to .Net Core supported list?

File: .github/ISSUE_TEMPLATE/Distribution_Request.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ assignees: ''
 
 - Name of the Distribution: 
 - Version of the Distribution:
+- Processor Architecture (One per request): 
 - [ ] The version of the Distribution is supported for at least one year.
 - [ ] The version of the Distribution is not an [interim release](https://ubuntu.com/about/release-cycle) or equivalent.
 - [ ] The version of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md#linux).
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Distribution_Request.md]:
```suggestion
- [ ] The version and architecture of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md#linux).
```

File: .github/ISSUE_TEMPLATE/Distribution_Request.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ assignees: ''
 - Processor Architecture (One per request): 
 - [ ] The version of the Distribution is supported for at least one year.
 - [ ] The version of the Distribution is not an [interim release](https://ubuntu.com/about/release-cycle) or equivalent.
-- [ ] The version of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md#linux).
+- [ ] The version and architecture of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md#linux).
```
Comment (iSazonov) [inline .github/ISSUE_TEMPLATE/Distribution_Request.md]: Why do we reference to Linux?

Comment (TravisEz13) [reply .github/ISSUE_TEMPLATE/Distribution_Request.md]: This is a distribution request... which implies Linux.  macOS... is macOS... Windows, we will automatically pickup new versions...

File: .github/ISSUE_TEMPLATE/Distribution_Request.md [hunk 4]
```diff
@@ -22,4 +23,4 @@ assignees: ''
 - [ ] Docker image published
 - [ ] Distribution tested
 - [ ] Lifecycle updated
-- [ ] Documentation Updated
+- [ ] Documentation Updated
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Distribution_Request.md]:
```suggestion
- [ ] Documentation Updated
```

**Review by iSazonov — APPROVED** [review]

**Pull request merged**